flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniClusterConfiguration.java

```diff
@@ -38,6 +38,8 @@
  */
 public class MiniClusterConfiguration {
 
+	static final String SCHEDULER_TYPE_KEY = "schedulerType";
```

Comment:
Not sure if there are any conventions about how to name system properties. Maybe this should be also named `jobmanager.scheduler` to keep it aligned with `JobManagerOptions.SCHEDULER`.

Reply:
No such convention yet. But I think it's fine to use the JobManagerOptions.SCHEDULER key as the property key as they take the same values.